docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,664 @@
+  - e.g. `ip link set dev cbr0 up`
+- See `cluster/saltbase/salt/_states/container_bridge.py`
+
+If you have turned off docker ip masquerading to allow pods to talk to each
+other, then you may need to do masquerading just for destination IPs outside
+the cluster network.  For example:
+```iptables -w -t nat -A POSTROUTING -o eth0 -j MASQUERADE \! -d ${CONTAINER_SUBNET}```
+will rewrite the source address from
+the PodIP to the Node IP for traffic bound outside the cluster, and kernel
+[connection tracking](http://www.iptables.info/en/connection-state.html#UDPCONNECTIONS)
+will ensure that responses destined to the node still reach
+the pod.
+
+#### Other configuration
+- Configure auto-upgrades on node, if desired.
+- Configure log rotation for all node components (e.g. using [logrotate](http://linux.die.net/man/8/logrotate)).
+- Setup liveness-monitoring (e.g. using [monit](http://linux.die.net/man/1/monit)).
+- Setup volume plugin support (optional)
+  - Install any client binaries for optional volume types, such as `glusterfs-client` for GlusterFS
+    volumes.
+
+#### Using Configuration Management
+The previous steps all involved "conventional" system administration techniques for setting up
+machines.  You may want to use a Configuration Management system to automate the node configuration
+process.  There are examples of Saltstack, Ansible, Juju, and CoreOS Cloud Config in the
+various Getting Started Guides.
+
+### Bootstrapping the Cluster
+
+While the basic node services (kubelet, kube-proxy, docker) are typically started and managed using
+traditional system administration/automation approaches, the remaining *master* components of Kubernetes are
+all configured and managed *by Kubernetes*:
+  - their options are specified in a Pod spec (yaml or json) rather than an /etc/init.d file or
+    systemd unit.
+  - they are kept running by Kubernetes rather than by init.
+  - they run in containers, and so tools like `ps` and `top` are less useful than `docker ps` and `kubectl get
+    pods`
+
+#### etcd
+You will need to run one or more instance of etcd.  
+  - Recommended approach: run one etcd instance, with its log written to a directory backed
+    by durable storage (RAID, GCE PD)
+  - Alternative: run 3 or 5 etcd instances.
+    - Log can be written to non-durable storage because storage is replicated. 
+    - run a single apiserver which connects to one of the etc nodes.
+ See [Availability](../availability.md) for more discussion on factors affecting cluster
+availability.
+
+To run an etcd instance:
+  - copy `cluster/saltbase/salt/etcd/etcd.manifest`
+  - make any modifications needed
+  - start the pod by putting it into the kubelet manifest directory
+
+#### Apiserver
+To run the apiserver:
+  - select the correct flags for your cluster
+  - write a pod spec for the apiserver using the provided template
+  - start the pod by putting it into the kubelet manifest directory
+
+Here are some apiserver flags you may need to set:
```

Comment:
Move this whole section about flags under the "select the correct flags for your cluster" (398), so these options show under the corresponding step in the process.